core/src/main/java/org/jdbi/v3/core/statement/Slf4JSqlLogger.java

```diff
@@ -0,0 +1,80 @@
+package org.jdbi.v3.core.statement;
+
+import java.sql.SQLException;
+import java.time.Duration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Simple {@link SqlLogger} that emits some diagnostic information about
+ * Jdbi usage. Usually you shouldn't log every database statement made but
+ * it can be helpful for debugging.
+ */
+public class Slf4JSqlLogger implements SqlLogger {
```

Comment:
consider final

Reply:
Why would you want to avoid people extending it?

Reply:
I don't think anyone should / will extend this, but there's also no particular reason to prevent it, and there's nothing more frustrating than absolutely needing to subclass a final class :)